requirements.txt

```diff
@@ -1,3 +1,5 @@
 mlflow>=2.0.0rc0
-ipykernel>=6.12.*
-ipython>=7.32.*
+scikit-learn>=1.1
+ipykernel>=6.12
+ipython>=7.32
+flaml
```

Comment:
Do we want to include this in the basic requirement? Specially given that none of the pipelines have an example for flaml? What do you think?

Reply:
Forgot to remove it. Was testing FLAML. It doesn't work unfortunately.